src/cookbook/navigation/set-up-universal-links.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Set up universal links for iOS
+description: How set up universal links for an iOS application built with Flutter
+prev:
+  title: Send data to a new screen
+  path: /cookbook/navigation/passing-data
+next:
+  title: Delete data on the internet
+  path: /cookbook/networking/delete-data
+js:
+- defer: true
+  url: https://dartpad.dev/inject_embed.dart.js
+---
+
+<?code-excerpt path-base="codelabs/deeplink_cookbook"?>
+
+Deep linking is a mechanism for launching an app with a URI. This URI
+contains scheme, host, and path, and opens the app to a specific
+screen.
+
+A _universal link_ is a type of deep link that uses `http` or `https` and
+is exclusive to Apple devices.
+
+
+Setting up universal links requires one to own a web domain. Otherwise, consider
+using [Firebase Hosting][] or [GitHub Pages][] as a temporary solution.
+
+
+## 1. Customize a Flutter application
+
+Write a Flutter app that can handle an incoming URL.
+This example uses the [go_router][] package to handle the routing.
+The Flutter team maintains the `go_router` package.
+It provides a simple API to handle complex routing scenarios.
+
+1. To create a new application, type `flutter create <app-name>`.
+
+    ```shell
+    $ flutter create deeplink_cookbook
+    ```
+
+2. To include `go_router` package in your app, add a dependency for
+`go_router` in the `pubspec.yaml` file.
+
+    ```yaml
+    dependencies:
+      flutter:
+        sdk: flutter
+      go_router: ^6.0.9
+    ```
+
+3. To handle the routing, create a `GoRouter` object in the `main.dart` file:
+
+    <?code-excerpt "lib/main.dart"?>
+    ```run-dartpad:theme-light:mode-flutter:run-true:width-100%:height-600px:split-60:ga_id-interactive_example
+    import 'package:flutter/material.dart';
+    import 'package:go_router/go_router.dart';
+
+    void main() => runApp(MaterialApp.router(routerConfig: router));
+
+    /// This handles '/' and '/details'.
+    final router = GoRouter(
+      routes: [
+        GoRoute(
+          path: '/',
+          builder: (_, __) => Scaffold(
+            appBar: AppBar(title: const Text('Home Screen')),
+          ),
+          routes: [
+            GoRoute(
+              path: 'details',
+              builder: (_, __) => Scaffold(
+                appBar: AppBar(title: const Text('Details Screen')),
+              ),
+            ),
+          ],
+        ),
+      ],
+    );
+    ```
+
+
+
+## 2. Adjust iOS build settings
+
+1. Launch Xcode.
+2. Open the `ios/Runner.xcworkspace` file inside the project’s `ios` folder.
+3. Navigate to the `Info` Plist file in the `ios/Runner` folder.
+
+    <img src="/assets/images/docs/cookbook/set-up-universal-links-info-plist.png" alt="Xcode info.Plist screenshot" width="100%" />
+
+4. In the `Info` property list, add a row.
+5. Set the key to `FlutterDeepLinkingEnabled` with a
```

Comment:
**question:** How do you set the key? Do you fill in a field? Which field?

Don't rely on the screenshot to communicate your ask. There could be folks that aren't using a visual browser or other issues, like language barriers. (Images can't be automatically translated.)

Reply:
added more details instructions